src/scripts.js

```diff
@@ -16,9 +19,27 @@ const favoriteRecipesSection = document.getElementById('favoriteRecipesSection')
 
 const weeklyRecipesSection = document.getElementById('weeklyRecipesSection');
 
+let searchBar = document.querySelector('.favorites-view__favorites');
+let mainGlide = document.querySelector('.search-ingredient-glide__images-container');
 
 homeViewButton.addEventListener('click', displayHomeView);
 allRecipesButton.addEventListener('click', displayAllRecipes);
+searchBar.addEventListener('keyup', (e) => {
+  mainGlide.innerHTML = '';
+  const searchTerm = e.target.value;
+  filteredRecipes = recipesList.filter(recipe => {
+    return recipe.name.toLowerCase().includes(searchTerm.toLowerCase());
+    || recipe.tags.toString().toLowerCase().includes(searchTerm.toLowerCase());
+    || recipe.ingredients.toString().toLowerCase().includes(searchTerm.toLowerCase());
```

Comment:
There's _a lot_ going on in this return statement that makes it hard to read. I would suggest setting all of these to variables and then returning something easier to read. Something like:

```javascript
const recipeIncludesName = recipe.name.toLowerCase().includes(searchTerm);
...

return recipeIncludesName || ... ;
```